src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java

```diff
@@ -250,8 +255,6 @@ public void onNeighborChanged() {
 
     @Override
     public void setFrontFacing(EnumFacing frontFacing) {
-        super.setFrontFacing(frontFacing);
-        this.didSetFacing = true;
```

Comment:
This change removes all setting of `didSetFacing`, which means that the check in `onNeighborChanged` will always fail